### DISCUSSION TITLE
Incorrect Sigma Rule to EQL Conversion in Security Onion

### DISCUSSION THREAD

**Discussion by medahalli** [Security-Onion-Solutions/securityonion#13293] [2.4]
### Version

2.4.80

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

6

### RAM

20 GO

### Storage for /

400

### Storage for /nsm

400

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hello,

I've encountered an issue with the conversion of Sigma rules to EQL queries in Security Onion. The conversion does not seem to work correctly for a specific rule.

![image (1)](https://github.com/Security-Onion-Solutions/securityonion/assets/82915199/5b3944e4-a9f2-4c2d-8eac-e5704e114b55)
![image (2)](https://github.com/Security-Onion-Solutions/securityonion/assets/82915199/adf03877-8ae0-45e8-bc3d-16495fd749e3)

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by defensivedepth** [comment]
Yes, so the short version is that this is a field mapping issue with some extra complexities. We are in the middle of fixing it and some others that have popped up.
